java.js

```diff
@@ -183,4 +185,25 @@ function checkLowerCase(input) {
 form.addEventListener('submit', (event) => {
   event.preventDefault();
   checkLowerCase(email);
-});
\ No newline at end of file
+});
+
+const valueInput = JSON.parse(localStorage.getItem('valueInput'));
+
+function addVale(n, e, t) {
+  valueInput.push({ n, e, t });
+  localStorage.setItem('valueInput', JSON.stringify(valueInput));
+  return { n, e, t };
+}
+
+function showNow({ n, e, t }) {
+  namee.value = n;
+  email.value = e;
+  text.value = t;
+}
+
+valueInput.forEach(showNow);
```

Comment:
- [ ] Kindly make sure the form data is persisted between refreshes as requested by the previous reviewer. 🤝️
This error is shown in the console, please take a look at it.

![Screenshot from 2022-12-23 20-59-13](https://user-images.githubusercontent.com/17243325/209386193-4e337e0f-d679-4f1e-a96f-a3daf7b38e21.png)

Reply:
Hello
Thank you for the feedback, but I don't understand. I have run this code in my browser, its not showing any [errors](![pp](https://user-images.githubusercontent.com/77361109/209391396-bee4ddc6-b2e0-4c1f-8f9a-3bb96dd655a9.PNG))